### ISSUE TITLE
Short Bans/ kick/ timeout

### ISSUE THREAD

**Issue by ToastyStoemp** [AndrewBelt/hack.chat#92]
Sometimes people are not actively breaking the rules. However are still disturbing the chat experience. I'd be convenient for moderators to have a, time based ban / kick / timeout function.

**Comment by M4GNV5** [comment]
a kick would be great

**Comment by 0x17de** [comment]
@AndrewBelt the kick is quite a must. theres some logging bot online atm that uploads backlog to some external webspace. a ban wouldn't even disable the logging functionality as he'd still be online. (Another issue: the bot's owner is not logged in at the moment or won't resolve the issue)

**Comment by AndrewBelt** [comment]
Should `ban` also kick?

**Comment by 0x17de** [comment]
@AndrewBelt hmm if so, it would be nice to have some reason shown on the banned client. otherwise we can't tell them what they did wrong

**Comment by AndrewBelt** [comment]
Don't moderators usually tell them before banning them?

**Comment by ToastyStoemp** [comment]
But if a bot is being punished, the owner of the bot can't necessarily understand the reason, while emitting a ban reason in the ban event, devs can patch the issues and get back with a updated version

**Comment by 0x17de** [comment]
@AndrewBelt yep that is correct. i thought of it being the last thing on the terminal like after they spammed the whole screen again.

**Comment by alienfxfiend** [comment]
All hell will break lose if this happens —too much power IMO.

**Comment by alienfxfiend** [comment]
All hell will break lose if this happens —too much power IMO. & everyone will want to be mod.
